libs/sdk-ui-kit/src/Header/Header.tsx

```diff
@@ -210,6 +211,32 @@ class AppHeaderCore extends Component<IAppHeaderProps & WrappedComponentProps, I
         onClick: this.state.responsiveMode && this.props.helpMenuItems ? this.toggleHelpMenu : undefined,
     });
 
+    private getTrialCountdown = (expiredDate: Date) => {
+        const trialDaysLeft = differenceInCalendarDays(expiredDate, new Date()) + 1; // include expired date.
```

Comment:
Can you try to get `differenceInCalendarMonths` first? if the result = 0, then we would get the `differenceInCalendarDays`.